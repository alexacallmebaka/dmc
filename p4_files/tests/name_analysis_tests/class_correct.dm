ClassA : class{
  a : perfect bool = true;
  print : (a: int) int {
    return a;
 }
};