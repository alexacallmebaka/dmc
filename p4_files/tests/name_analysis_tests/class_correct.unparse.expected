ClassA{ClassA} : class {
    a{perfect bool} : perfect bool = true;
    print{(int)->int} : (a{int} : int) int  {
        return a{int};
    }
};